<?xml version="1.0"?>
<launch>
	<!-- launch differential -->
	<node pkg="zuros_motor_transformation" type="motor_transformation_differential" name="motor_transformation_differential_node" output="screen">
    <param name="wheel_diameter" value="0.3" />
    <param name="wheel_base" value="0.54" />
  </node>
</launch>
